packages/bot/src/components/toolbar.jsx

```diff
@@ -118,16 +119,12 @@ const BotNameBox = ({ onBotNameTyped, file_name }) => (
     </div>
 );
 
-const ButtonGroup = ({
-    is_stop_button_disabled,
-    is_stop_button_visible,
+const WorkspaceGroup = ({
```

Comment:
I  agree with renaming, but I think something like `ToolbarGroup` makes more sense as this component is unrelated to the workspace.